content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-kubeconfig.md

```diff
@@ -0,0 +1,24 @@
+---
+title: kubeadm kubeconfig
+content_type: conceito
+weight: 90
+---
+
+`kubeadm kubeconfig` fornece utilitários para gerenciar arquivos kubeconfig.
+
+Exemplos de como usar `kubeadm kubeconfig user`, consulte [Gerando arquivos kubeconfig para usuários adicionais](/docs/tasks/administer-cluster/kubeadm/kubeadm-certs#kubeconfig-additional-users).
+
+
+## kubeadm kubeconfig {#cmd-kubeconfig}
+
+{{< tabs name="tab-kubeconfig" >}}
+{{< tab name="visão geral" include="generated/kubeadm_kubeconfig.md" />}}
```

Comment:
You have to copy `content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_kubeconfig.md` to `content/pt-br` as well, or it will cause build fail.

Reply:
Hi @Sea-n Thank you so much for trying to help me. I saw this error and I'm not sure what it is. It's a little odd that I copy to a different folder than the original folder. Are you sure about that? I'm thinking to  closing all PR that failed.

Reply:
Hey @MrErlison, the `include="path/to/xxx"` statement is using relative path which refer to `content/pt-br/docs/reference/setup-tools/kubeadm/path/to/xxx`, this made translating code comments possible.

BTW, maybe you would like to join `#sig-docs` channel in [Kubernetes Slack](https://slack.k8s.io/).
It's not that efficient to discuss on GitHub comment.

Reply:
Thank you. /unhold